.github/workflows/coverage.yml

```diff
@@ -48,8 +48,8 @@ jobs:
     steps:
       - uses: actions/setup-go@v4
         with:
-          go-version: "1.22.2"
-      - uses: actions/checkout@v4
+          go-version: "1.22.5"
+      - uses: actions/checkout@v3
```

Comment:
this is getting downgraded here. would it make sense to have a follow up PR to bump?

Reply:
makes sense